partner_fax/__init__.py

```diff
@@ -0,0 +1,4 @@
+# © 2018 Apruzzese Francesco <f.apruzzese@apuliasoftware.it>
```

Comment:
Please take the opportunity to replace the copyright simbol by `Copyright` according to https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L2

Reply:
I dropped the line from this file since this is an __init__.py

Reply:
right!